assets/stylesheets/_components.scss

```diff
@@ -11,7 +11,7 @@
 @import 'auth/style';
 @import 'components/accordion/style';
 @import 'components/app-promo/style';
-@import 'components/author-selector/style';
+@import 'blocks/author-selector/style';
```

Comment:
I will certainly do it once I clean up the folders a bit more. I may separate the scss imports for blocks and components as well, or group them differently.